source/main/utils/blockSyncProgressHelpers.ts

```diff
@@ -1,8 +1,8 @@
 import moment, { Moment } from 'moment';
 
 export function isItFreshLog(applicationStartDate: Moment, line: string) {
-  const [, logDate] = line.match(
-    /\[(\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}\.\d+) UTC]/
-  );
-  return applicationStartDate.isBefore(moment.utc(logDate));
+  const [, logDate] =
+    line.match(/\[(\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}\.\d+) UTC]/) || [];
+
+  return !!logDate && applicationStartDate.isBefore(moment.utc(logDate));
```

Comment:
I found `!!` a bit hard to understand. Maybe we could change to something else? 

```suggestion
if(!logDate) {
return false;
}

  return applicationStartDate.isBefore(moment.utc(logDate));
```

Reply:
Updated, thanks!